src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -5774,6 +5849,10 @@ public override BoundNode VisitParameter(BoundParameter node)
             int slot = GetOrCreateSlot(parameter);
             var type = GetDeclaredParameterResult(parameter);
             SetResult(node, GetAdjustedResult(type, slot), type);
+            if (_reinferredVariablesOpt?.GetReinferredSymbol(node.ParameterSymbol) is ParameterSymbol updatedParameter)
```

Comment:
>(_reinferredVariablesOpt?.GetReinferredSymbol(node.ParameterSymbol) is ParameterSymbol updatedParameter [](start = 15, length = 103)

I don't believe I saw where we add this information to the cache.

Reply:
No longer relevant.

---
In reply to: [327924175](https://github.com/dotnet/roslyn/pull/38769#discussion_r327924175) [](ancestors = 327924175)